tests/terratest/components/aksClusterSpecs1.go

```diff
@@ -0,0 +1,5 @@
+package components
+
+var AKSClusterSpecs1 = `"
```

Comment:
are all these generated by go + TF? If not, why can't all these vars be in one file, say per provider?

Reply:
These were tricky.  These were all created by me, and allows Go to build out the modules main.tf file. 

All files located in the components folder, hold Terraform Rancher2 provider resource blocks.  When SetConfigTF() is called, depending on the module, ultimately a giant string literal is composed of select files from the components folder, are built out and written to modules main.tf file.

In some cases, the entire block was able to be grabbed, and naming convention was appropriate (e.g. `provider.go`)  For some, I needed to break the blocks up, to interpolate values to allow them to be updated.  Those values come from the cattle-config.yaml.  In these instances, I didn't know what to call the blocks, as i tried break them up minimally, so used a number system.

Suggestions to improve this would be appreciated.

Reply:
Ahhh, re-reading your initial comment, these are only grabbed and referenced by go, so yes, we absolutely could organize this better and place them in files per provider. 👍 Great idea!